src/assets/javascript/analytics/cookies.js

```diff
@@ -0,0 +1,58 @@
+/* global document window */
+
+window.DI = window.DI || {};
+
+(function(DI){
+
+  'use strict'
+
+  const cookies = {
+
+    hasConsentForAnalytics: function () {
+      const COOKIES_PREFERENCES_SET = "cookies_preferences_set";
```

Comment:
This same const is defined in a few places, is it possible to put it in a central place where it can be reused?

Reply:
Actually there are a few things that are reused like the `getCookie` function, can we share those too?

Reply:
Ah my code search is including `dist` so it looks like there is double of everything. All good leave it as is